modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -362,6 +362,34 @@ test.describe('Manage object definitions through Model Builder', () => {
 			objectDefinition2.id
 		);
 	});
+
+	test('delete object via objects card', async ({
+		apiHelpers,
+		modelBuilderPage,
+	}) => {
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				status: {code: 0},
+			});
+
+		await modelBuilderPage.goto({objectFolderName: 'Default'});
+
+		await modelBuilderPage.leftSidebarItems
+			.filter({hasText: objectDefinition.name})
+			.click();
+
+		await modelBuilderPage.clickObjectDefinitionActionsButton(
+			objectDefinition.name
+		);
+
+		await modelBuilderPage.deleteObjectDefinition(objectDefinition.name);
+
+		await expect(
+			modelBuilderPage.objectDefinitionNodes.filter({
+				hasText: objectDefinition.name,
+			})
+		).toBeHidden();
+	});
```

Comment:
Hey Rafa, I would prefer to use `not.toBeVisible()` in this case, the difference between them is:

`toBeHidden()`: checks if an element is present but hidden.
`not.toBeVisible()`: checks if an element is not visible in any way.

Since you want to check if the object definition is not present in the structure because it was deleted, I think it's better to use the `not.toBeVisible()` :smile: 

One more thing, to check if the object definition has been deleted, you need to check if it has also been removed from the left sidebar, so you will need to add one more expect in this test:

```ts
await expect(
	modelBuilderPage.leftSidebarItems
	.filter({hasText: objectDefinition.name})
).not.toBeVisible();
```

Reply:
Thank you very much for the explanation! It's clearer now the difference between the functions! I've fixed it!